src/components/select/Select.js

```diff
@@ -421,7 +421,7 @@ export default class SelectComponent extends ListComponent {
     }
 
     // Add the value options.
-    if (!fromSearch) {
+    if (!fromSearch || this.component.widget === 'html5') {
```

Comment:
Why is ```fromSearch``` true for html5 widget settings?  Wouldn't that be the the source of the real problem instead of fixing it with a bandaid?